getting-started/002-search.md

```diff
@@ -90,19 +91,17 @@ Results in the United States:
 * YMCA, Bremerton, WA
 * YMCA, Westerly, RI
 
-
-### ...to a rectangular region
+### Search within a rectangular region
 
 ![](https://github.com/dianashk/pelias-doc/blob/master/getting-started/world_rect.png)
- 
-In the case where you need to specify the boundary using a rectangle, all we need is a pair of coordinates on earth. Here are a few examples:
 
-#### Example time
-Let's say you wanted to find museums in the state of **Texas**. You'd need to set the `boundary.rect.*` parameter grouping to values representing the bounding box around **Texas**: min_lon=-106.65 min_lat=25.84 max_lon=-93.51 max_lat=36.5
+To specify the boundary using a rectangle, you need latitude, longitude coordinates for two corners of the bounding box (the mininum and the maximum latitude, longitude).
```

Comment:
Was trying to figure out how to say this -- as a coordinate pair is one set of x,y, and you need two of them. Will change to diagonal.